README.md

```diff
@@ -126,7 +126,7 @@ And tell `webpack` to generate sourcemaps.
 ```js
 webpack: {
   // ...
-  devtool: 'inline-source-map';
+  'devtool': 'inline-source-map'
```

Comment:
The `''`'s shouldn't be needed here!

Reply:
This is needed because of the linter rules of this project, if you remove the single quotes the CI lint will fail.